pxd.c

```diff
@@ -1132,7 +1132,7 @@ int pxd_init(void)
 out_misc:
 	misc_deregister(&pxd_miscdev);
 out_fuse:
-	for (j = 0; j < i; ++j) {
+	for (j = 0; j <= i; ++j) {
```

Comment:
I think problem still exists. If pxd_context_setup() fails at i, we cant call destroy for i.